### PR TITLE
Add design for a referee feedback form after giving a reference

### PR DESCRIPTION
**Pull request by fofr**
localhost_3000_referee-confirmation (2)
localhost_3000_referee-confirmation (1)

https://trello.com/c/aaKMc14Y/800-design-questionnaire-to-get-feedback-from-referees

### CODE REVIEW

**Comment by fofr** [comment]
For reviewers: Markup has problems with the repeated conditional HTML fields, but as this is an illustrative design that doesn't fit into a user journey within the prototype, I figured this would be fine.

**Comment by adamsilver** [comment]
Nice one @fofr. In addition to what I said on Slack I'd suggest:

- not using the word ‘please’
- using text labels instead of numbers (and not a mix)
- looking if we could combine the heading and paragraph to something like ‘Tell us about your experience so we can improve the way giving a reference works’
- considering marking (some of) the questions as optional

**Comment by fofr** [comment]
> not using the word ‘please’
Discussed this with @emmajhf  and Kahar, and we think this is a good exception to https://www.gov.uk/service-manual/design/writing-for-user-interfaces#tone because this is an optional survey and only beneficial to us.

> using text labels instead of numbers (and not a mix)
Good idea. @emmajhf what words would you suggest? (Very poor, Poor, Ok, Good, Very good?)

File: app/views/referee-confirmation.njk [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+{% block content %}
+  {{ govukPanel({
+    classes: "govuk-!-margin-bottom-8",
+    titleHtml: "Your reference for Jane Doe has been&nbsp;submitted"
+  }) }}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-heading-l">Tell us about your experience giving a&nbsp;reference</h2>
```
Comment (fofr) [inline app/views/referee-confirmation.njk]:
```suggestion
      <h2 class="govuk-heading-l">Tell us about your experience of giving a&nbsp;reference</h2>
```

File: app/views/referee-confirmation.njk [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        } | decorateApplicationAttributes(["referee-confirmation"])) }}
+      {% endset %}
+
+      {% set whyGuidanceRatingHtml %}
+        {{ govukTextarea({
+          label: {
+            text: "Tell us why you chose that rating"
+          },
+          hint: {
+            text: "Did you have questions that weren’t answered by the guidance?"
```
Comment (fofr) [inline app/views/referee-confirmation.njk]:
```suggestion
            text: "Did you have questions that were not answered by the guidance?"
```

**Comment by emmajhf** [comment]
@fofr, I think those labels (Very poor, Poor, Ok, Good, Very good) work well. I agree about not needing numbers.

**Pull request merged**